test/NuGet.Core.Tests/NuGet.CommandLine.Xplat.Tests/compiler/resources/Test.NoPackageRidSpecificPackageProject.project.assets.json

```diff
@@ -1,455 +0,0 @@
-{
-  "version": 3,
-  "targets": {
-    "net9.0": {
-      "Microsoft.NETCore.Platforms/1.1.0": {
-        "type": "package",
-        "compile": {
-          "lib/netstandard1.0/_._": {}
-        },
-        "runtime": {
-          "lib/netstandard1.0/_._": {}
-        }
-      },
-      "Microsoft.NETCore.Targets/1.1.0": {
-        "type": "package",
-        "compile": {
-          "lib/netstandard1.0/_._": {}
-        },
-        "runtime": {
-          "lib/netstandard1.0/_._": {}
-        }
-      },
-      "System.AppContext/4.3.0": {
-        "type": "package",
-        "dependencies": {
-          "System.Runtime": "4.3.0"
-        },
-        "compile": {
-          "ref/netstandard1.6/System.AppContext.dll": {
-            "related": ".xml"
-          }
-        },
-        "runtime": {
-          "lib/netstandard1.6/System.AppContext.dll": {}
-        }
-      },
-      "System.Runtime/4.3.0": {
-        "type": "package",
-        "dependencies": {
-          "Microsoft.NETCore.Platforms": "1.1.0",
-          "Microsoft.NETCore.Targets": "1.1.0"
-        },
-        "compile": {
-          "ref/netstandard1.5/System.Runtime.dll": {
-            "related": ".xml"
-          }
-        }
-      }
-    },
-    "net9.0/linux-x64": {
-    },
```

Comment:
I deleted this file and the test that uses it, because RID specific targets still use the same top level package as the RIDless target, so it doesn't make sense for the rid specific target to not have the direct packages that the project defines under `$.project.frameworks`